src/main/java/com/shootoff/camera/cameratypes/PS3EyeCamera.java

```diff
@@ -613,7 +577,7 @@ protected ps3eye_t(Pointer ps3eye_t) {
 		 **/
 		int ps3eye_get_parameter(ps3eye_t eye, int ps3eyeGain);
 
-		// a test returns 85
+		// a test inserted by ifly53e returns 85
```

Comment:
Since this isn't used, and presumably you no longer need it for testing, please remove this function definition.